resources/public/api/definition.json

```diff
@@ -15,7 +15,30 @@
       {
         "version": "1.0",
         "status": "BETA",
-        "endpointsEnabled": true
+        "endpointsEnabled": true,
+        "access": {
+          "type": "PUBLIC"
+        },
```

Comment:
We don't need to explicitly state the access here because PUBLIC is the default. See [here](https://github.com/hmrc/api-publisher/blob/main/docs/api-definition.md#versions)